toqito/nonlocal_games/binary_constraint_system_game.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+import networkx as nx
+from toqito.nonlocal_games.nonlocal_game import NonlocalGame
+
+def bcs(M, b):
```

Comment:
I think this function should be renamed to something other than `bcs`. Is there not some way to rectify the way in which a BCS game is constructed from the constraint list within the `nonlocal_game.py` file and this function?